### PR TITLE
Mobile tests

### PR DESCRIPTION
**Pull request by BelyukMihail**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenAndroid.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.qaprosoft.carinademo.mobile.common;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class SauceDemoAbstractScreenAndroid extends SauceDemoAbstractScreenBase {
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenAndroid.java]: 1. rename to SauceDemoAbstractScreen
2. move to Android folder

File: src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenBase.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.openqa.selenium.WebDriver;
+
+public abstract class SauceDemoAbstractScreenBase extends AbstractPage {
+
+    public SauceDemoAbstractScreenBase(WebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isOpened() {
+        throw new UnsupportedOperationException();
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenBase.java]: Add some messaging. For example you can move it to iOS and Android classes and add messages 
UnsupportedOperationException("Not implemented for iOS");
UnsupportedOperationException("Not implemented for Android");

Comment (ankochergin) [reply src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenBase.java]: Also messages can be moved to constants

File: src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenAndroid.java (hunk 1 above)
Comment (ankochergin) [reply src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenAndroid.java]: missing annotation

File: src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenIOS.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.qaprosoft.carinademo.mobile.common;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class SauceDemoAbstractScreenIOS extends SauceDemoAbstractScreenBase {
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenIOS.java]: missing annotation

Comment (ankochergin) [reply src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenIOS.java]: rename to SauceDemoAbstractScreen

File: src/main/resources/_config.properties [hunk 4]
```diff
@@ -45,35 +45,36 @@ retry_count=0
...
-#capabilities.udid=
-#capabilities.app=https://qaprosoft.s3-us-west-2.amazonaws.com/carinademoexample.zip
+selenium_url=http://localhost:4723/wd/hub
+capabilities.platformName=iOS
+capabilities.deviceName=iPhone 14 pro
+capabilities.deviceType=phone
+capabilities.platformVersion=16.2
+capabilities.automationName=XCUITest
+capabilities.udid=A2F161D6-F506-40B4-BDAB-55095BC2CAC3
+capabilities.app=https://github.com/saucelabs/sample-app-mobile/releases/download/2.7.1/iOS.Simulator.SauceLabs.Mobile.Sample.app.2.7.1.zip
```
Comment (ankochergin) [inline src/main/resources/_config.properties]: you can download it on your computer and set local path

File: src/main/resources/_config.properties [hunk 5]
```diff
@@ -45,35 +45,36 @@ retry_count=0
...
 #======== Local Run for iOS Mobile ===============#
-#selenium_url=http://localhost:4723/wd/hub
-#capabilities.platformName=iOS
-#capabilities.deviceName=<simulator_name>
-#capabilities.deviceType=phone
-#capabilities.platformVersion=<simulator_platform_version>
-#capabilities.automationName=XCUITest
-#capabilities.udid=
-#capabilities.app=https://qaprosoft.s3-us-west-2.amazonaws.com/carinademoexample.zip
+selenium_url=http://localhost:4723/wd/hub
```
Comment (ankochergin) [inline src/main/resources/_config.properties]: don't commit your local configs

File: src/test/java/com/qaprosoft/carinademo/mobile/AuthenticationTest.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import com.qaprosoft.carinademo.mobile.common.LoginScreenBase;
+import com.qaprosoft.carinademo.mobile.ios.LoginScreen;
+import com.zebrunner.carina.utils.R;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AuthenticationTest extends SauceAppTest {
+
+    @Test
+    public void openAppTest() {
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/AuthenticationTest.java]: I would say better name is loginTest

File: src/test/resources/suite/sauce-suite.xml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Sauce Suite" parallel="methods" thread-count="16">
+    <test name="All tests">
+        <classes>
+            <class name="com.qaprosoft.carinademo.web.AuthenticationTest"/>
+            <class name="com.qaprosoft.carinademo.web.CheckoutTest"/>
+            <class name="com.qaprosoft.carinademo.web.MenuTest"/>
+            <class name="com.qaprosoft.carinademo.web.ProductTest">
+                <methods>
+                    <exclude name="compareItemDescriptionTest"/>
```
Comment (ankochergin) [inline src/test/resources/suite/sauce-suite.xml]: I don't see this method in your test class

Comment (BelyukMihail) [reply src/test/resources/suite/sauce-suite.xml]: It's one of the ProductTest.class in  web  package

File: src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutCompleteScreen.java [hunk 8]
```diff
@@ -0,0 +1,25 @@
+package com.qaprosoft.carinademo.mobile.android;
+
+import com.qaprosoft.carinademo.mobile.common.CheckoutCompleteScreenBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import org.openqa.selenium.WebDriver;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = CheckoutCompleteScreenBase.class)
+public class CheckoutCompleteScreen extends CheckoutCompleteScreenBase {
+
+    private static final String THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID = "This method is not implemented on ANDROID";
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutCompleteScreen.java]: move this to SauceDemoAbstractScreenBase to avoid code duplication

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    @Override
+    public CheckoutCompleteScreenBase clickFinishBtn() {
+        finishBtn.click();
+        return initPage(getDriver(), CheckoutCompleteScreenBase.class);
+    }
+
+    @Override
+    public boolean isOpened() {
+        throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_IOS);
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java]: this should be in SauceDemoAbstractScreen. in inherited screens you override it (like you did in HomeScreen)

File: src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+        checkoutStepOne.enterFirstName(firstName);
+        checkoutStepOne.enterLastName(lastName);
+        checkoutStepOne.enterPostalCode(postalCode);
+
+        String firstNameFieldContent = checkoutStepOne.getFirstNameInput();
+        String lastNameFieldContent = checkoutStepOne.getLastNameInput();
+        String postalCodeFieldContent = checkoutStepOne.getPostalCodeInput();
+        Assert.assertEquals(firstName, firstNameFieldContent, "Field content doesn't match input data.");
+        Assert.assertEquals(lastName, lastNameFieldContent, "Field content doesn't match input data.");
+        Assert.assertEquals(postalCode, postalCodeFieldContent, "Field content doesn't match input data.");
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java]:
```suggestion
        Assert.assertEquals(firstName, checkoutStepOne.getFirstNameInput(), "Field content doesn't match input data.");
        Assert.assertEquals(lastName, checkoutStepOne.getLastNameInput(), "Field content doesn't match input data.");
        Assert.assertEquals(postalCode, checkoutStepOne.getPostalCodeInput(), "Field content doesn't match input data.");
```

File: src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+        CheckoutStepTwoScreenBase checkoutStepTwo = checkoutStepOne.clickContinueBtn();
+        CheckoutCompleteScreenBase checkoutComplete = checkoutStepTwo.clickFinishBtn();
+        Assert.assertTrue(checkoutComplete.isOpened(), "Failed to navigate to 'Checkout complete' screen");
+        String message = checkoutComplete.getCheckoutCompleteMessage();
+        Assert.assertEquals(message, expectedMessage);
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java]:
```suggestion
        Assert.assertEquals(checkoutComplete.getCheckoutCompleteMessage(), expectedMessage);
```

File: src/test/java/com/qaprosoft/carinademo/mobile/SauceAppTest.java [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+import com.qaprosoft.carinademo.util.CountMobileService;
+import com.qaprosoft.carinademo.util.ValueGeneratorService;
+
+public class SauceAppTest implements IAbstractTest {
+
+    protected AuthMobileService authService = new AuthMobileService();
+    protected CountMobileService countService = new CountMobileService();
+    protected AddProductsToCartMobileService addToCartService = new AddProductsToCartMobileService();
+    protected int cartXcoordinate = 340;
+    protected int cartYcoordinate = 70;
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/SauceAppTest.java]: move this to page object. also it would be great to calculate coordinates instead of hardcoding

File: src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutStepOneScreen.java [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+    @Override
+    public String getPostalCodeInput() {
+        throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID);
+    }
+
+    @Override
+    public boolean isOpened() {
+        throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID);
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutStepOneScreen.java]: you can remove it from here because it's already is in SauceDemoAbstractScreen
Override it only when you want to implement it
Also usually isOPened method is written after constructor

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java]: override it only if you want to implement it, it already throw exception in SauceDemoAbstractScreen

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/HomeScreen.java [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+        int cartXcoordinate = 340;
+        int cartYcoordinate = 70;
+        tap(cartXcoordinate, cartYcoordinate);
+        return initPage(getDriver(), ShoppingCartScreenBase.class);
+    }
+
+    @Override
+    public List<ExtendedWebElement> getProducts() {
+        return addToCartBtns;
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/HomeScreen.java]: !never return elements from page objects!

File: src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ProductsTest extends SauceAppTest {
+
+    @Test
+    public void sortProductPricesFromLowToHighTest() {
+        HomeScreenBase homePage = authService.authenticate(R.TESTDATA.get("user_name_good"), R.TESTDATA.get("password"));
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java]: lets create in AUthServie method authenticateBaseUser() to remove `R.TESTDATA.get("user_name_good"), R.TESTDATA.get("password")` in each test

File: src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+import com.qaprosoft.carinademo.util.ValueGeneratorService;
+import com.zebrunner.carina.utils.R;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CheckoutTest extends SauceAppTest {
+
+    @Test
+    public void checkIfUserCanCheckoutTest() {
+        int[] productsCount = {1, 2};
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/CheckoutTest.java]: provide not array, but just int value

File: src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+        homePage.clickPriceLowToHigh();
+        List<Double> prices = homePage.getProductPrices().stream()
+                .map(product -> Double.parseDouble(product.replace("$", "")))
+                .collect(Collectors.toList());
+        Assert.assertTrue(Comparators.isInOrder(prices, Comparator.naturalOrder()), "Products are not sorted by Price(low to high).");
+    }
+
+    @Test
+    public void deleteProductFromCartUsingSwipeTest() {
+        int[] productCounts = {1, 2, 3};
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java]: same here

File: src/main/java/com/qaprosoft/carinademo/util/CountMobileService.java [hunk 19]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.qaprosoft.carinademo.mobile.common.ShoppingCartScreenBase;
+import com.zebrunner.carina.utils.factory.ICustomTypePageFactory;
+
+public class CountMobileService implements ICustomTypePageFactory {
+
+    public int countProductsInShoppingCart() {
+        ShoppingCartScreenBase shoppingCartScreen = initPage(getDriver(), ShoppingCartScreenBase.class);
+        return shoppingCartScreen.getProductList().size();
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/util/CountMobileService.java]: I think better to have getProductCount in ShoppingCardScreen page object

File: src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+        HomeScreenBase homeAbstractPage = authService.authenticate(R.TESTDATA.get("user_name_good"), R.TESTDATA.get("password"));
+        addToCartService.addProductsToCart(productCounts);
+        ShoppingCartScreenBase shoppingCart = homeAbstractPage.clickShoppingCartBtn();
+        int productInCartBeforeDelete = countService.countProductsInShoppingCart();
+        shoppingCart.deleteProductsFromCartBySwipe();
+        int productsInCartAfterDelete = countService.countProductsInShoppingCart();
+        Assert.assertTrue(productInCartBeforeDelete == productCounts.length && productsInCartAfterDelete == 0,
```
Comment (ankochergin) [inline src/test/java/com/qaprosoft/carinademo/mobile/ProductsTest.java]: split into 2 separate asserts

File: src/main/java/com/qaprosoft/carinademo/util/AuthMobileService.java [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+
+    public HomeScreenBase login(String userName, String password) {
+        LoginScreenBase loginPage = initPage(getDriver(), LoginScreenBase.class);
+        return loginPage.login(userName, password);
+    }
+
+    public HomeScreenBase loginBaseUser() {
+        LoginScreenBase loginPage = initPage(getDriver(), LoginScreenBase.class);
+        return loginPage.login(R.TESTDATA.get("user_name_good"), R.TESTDATA.get("password"));
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/util/AuthMobileService.java]: reuse login method

File: src/main/java/com/qaprosoft/carinademo/web/page/HomePage.java [hunk 22]
```diff
@@ -47,4 +47,6 @@ public void clickMenuBtn() {
     public MenuBar getMenuBar() {
         return menuBar;
     }
+
+
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/web/page/HomePage.java]: is this change needed?

File: src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutCompleteScreen.java [hunk 23]
```diff
@@ -0,0 +1,23 @@
...
+
+    @Override
+    public String getCheckoutCompleteMessage() {
+        throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID);
+    }
+
+    @Override
+    public boolean isOpened() {
+        throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID);
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/android/CheckoutCompleteScreen.java]: do you still need isOpened methods if you already throw exceptions in abstract page?

File: src/main/java/com/qaprosoft/carinademo/mobile/common/HomeScreenBase.java [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+
+    public abstract List<String> getProductPrices();
+
+    public abstract ShoppingCartScreenBase clickShoppingCartBtn();
+
+    public abstract void addProductsToCart(int productCount);
+
+}
+
+
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/HomeScreenBase.java:25]: empty lines

File: src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenBase.java [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+
+    public boolean isOpened() {
+        DeviceType.Type deviceType = PlatformService.getDeviceInformation();
+        switch (deviceType) {
+            case IOS_PHONE:
+                throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_IOS);
+            case ANDROID_PHONE:
+                throw new UnsupportedOperationException(THIS_METHOD_IS_NOT_IMPLEMENTED_ON_ANDROID);
+            default:
+                return false;
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/SauceDemoAbstractScreenBase.java]: anyway we should throw an exception

File: src/main/java/com/qaprosoft/carinademo/mobile/common/ShoppingCartScreenBase.java [hunk 26]
```diff
@@ -0,0 +1,22 @@
...
+        super(driver);
+    }
+
+    public abstract List<ExtendedWebElement> getProductList();
+
+    public abstract void deleteProductsFromCartBySwipe();
+
+    public abstract CheckoutStepOneScreenBase clickCheckoutBtn();
+
+    public abstract int countProductsInCart();
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/common/ShoppingCartScreenBase.java]: getProductsInCardCount()

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutCompleteScreen.java [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+    private ExtendedWebElement checkoutCompleteMessage;
+
+    public CheckoutCompleteScreen(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public String getCheckoutCompleteMessage() {
+        return checkoutCompleteMessage.getText();
+    }
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutCompleteScreen.java]: you already check label in locator
so it would be enough to have method isCheckoutCompleteMessageShown()

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/LoginScreen.java [hunk 28]
```diff
@@ -0,0 +1,48 @@
...
+public class LoginScreen extends LoginScreenBase {
+
+    @ExtendedFindBy(iosClassChain = "**/XCUIElementTypeTextField[`name == 'test-Username'`]")
+    private ExtendedWebElement userNameField;
+
+    @ExtendedFindBy(iosClassChain = "**/XCUIElementTypeSecureTextType[`name='Password'`]")
+    private ExtendedWebElement passwordField;
+
+    @ExtendedFindBy(iosClassChain = "**/XCUIElementTypeOther[`name='test-LOGIN'`]")
+    private ExtendedWebElement loginBtn;
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/LoginScreen.java]: be consistent when writing locators. some spaces are missing between words and equal signs

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java (hunk 14 above)
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/CheckoutStepTwoScreen.java]: implement it or remove

File: src/main/java/com/qaprosoft/carinademo/util/PlatformService.java [hunk 29]
```diff
@@ -0,0 +1,21 @@
+package com.qaprosoft.carinademo.util;
+
+import com.qaprosoft.carina.core.foundation.webdriver.device.Device;
+import com.zebrunner.carina.utils.factory.DeviceType;
+
+public class PlatformService {
+
+    public static final String IPHONE_FOURTEEN = "iphone 14";
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/util/PlatformService.java]: let create enum for devices

File: src/main/java/com/qaprosoft/carinademo/util/PlatformService.java [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+    public static DeviceType.Type getDeviceType() {
+        return Device.currentDevice.get().getDeviceType();
+    }
+
+    public static String getDeviceName() {
+        return Device.currentDevice.get().getName();
+    }
+
+    public static boolean isDevice(String deviceName) {
+        return getDeviceName().toLowerCase().contains(deviceName);
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/util/PlatformService.java]: `contains` might not work correctly because we can have `iPhone 14`, `iPhone 14 Pro`, `iPhone 14 Pro Max` and all of them contain `iPhone 14`

File: src/main/java/com/qaprosoft/carinademo/mobile/ios/HomeScreen.java [hunk 31]
```diff
@@ -0,0 +1,81 @@
...
+                .map(ExtendedWebElement::getText)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public ShoppingCartScreenBase clickShoppingCartBtn() {
+        if (PlatformService.isDevice(PlatformService.IPHONE_FOURTEEN)) {
+            Dimension dimension = shoppingCartBtn.getSize();
+            Point point = shoppingCartBtn.getLocation();
+            tapElementBottomLeft(point.x, point.y, dimension.height);
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/ios/HomeScreen.java]: I think you can just write here `tap(point.x + 1, point.y + dimension.height - 1);` instead of new method

File: src/main/java/com/qaprosoft/carinademo/mobile/DeviceName.java [hunk 32]
```diff
@@ -0,0 +1,16 @@
+package com.qaprosoft.carinademo.mobile;
+
+public enum DeviceName {
+
+    IPHONE_14_PRO("iphone 14 pro");
```
Comment (ankochergin) [inline src/main/java/com/qaprosoft/carinademo/mobile/DeviceName.java]: I think it's case sensitive

File: src/main/resources/_config.properties [hunk 33]
```diff
@@ -1,121 +0,0 @@
-
```
Comment (ankochergin) [inline src/main/resources/_config.properties]: config is now completely deleted

**Review by ankochergin — COMMENTED** [review]
+ create xml suite for mobile tests

File: pom.xml [hunk 34]
```diff
@@ -129,7 +130,8 @@
 					<argLine>-javaagent:${project.build.directory}/agent/zebrunner-agent.jar</argLine>
 					<useSystemClassLoader>false</useSystemClassLoader>
 					<suiteXmlFiles>
-						<suiteXmlFile>${project.build.directory}/test-classes/testng_suites/${suite}.xml</suiteXmlFile>
+						<suiteXmlFile>${project.build.directory}/test-classes/suite/${suite-web}.xml</suiteXmlFile>
+						<suiteXmlFile>${project.build.directory}/test-classes/${suite-mobile}.xml</suiteXmlFile>
```
Comment (ankochergin) [inline pom.xml]:
```suggestion
						<suiteXmlFile>${project.build.directory}/test-classes/suite/${suite}.xml</suiteXmlFile>
```

File: pom.xml [hunk 35]
```diff
@@ -17,7 +17,9 @@
 		<carina-aws-s3.version>1.0.0</carina-aws-s3.version>
 		<java.version>11</java.version>
 		<zebrunner-agent.version>1.9.2</zebrunner-agent.version>
-		<suite>helloWorld</suite>
+		<suite-web>sauce-suite-web</suite-web>
+		<suite-mobile>sauce-suite-mobile</suite-mobile>
+		<suite-api>api-suite</suite-api>
```
Comment (ankochergin) [inline pom.xml]:
```suggestion
		<suite>api-suite</suite>
```
